packages/eui/src/components/flyout/flyout.tsx

```diff
@@ -38,8 +38,22 @@ export type EuiFlyoutProps<T extends ElementType = 'div' | 'nav'> = Omit<
   EuiFlyoutComponentProps<T>,
   'as'
 > & {
+  /**
+   * Enable session management to create flyout compositions and journeys
+   * like side-to-side flyout rendering based on parent-child grouping,
+   * simple flyout transitions, history, and more.
+   *
+   * Check out [EuiFlyout session management](https://eui.elastic.co/docs/components/containers/flyout/session-management)
+   * to learn more.
+   */
   session?: boolean;
```

Comment:
This line was changed in the upstream:
```suggestion
  session?:
    | typeof SESSION_START
    | typeof SESSION_INHERIT
    | typeof SESSION_NEVER;
```